### ISSUE TITLE
assignVariables - use base jsonData as variable

### ISSUE THREAD

**Issue by Arch0rr** [apideck-libraries/portman#429]
Hello!
When assigning variables, I'd like to be able to use the responseJson as the variable itself, as the response isn't an object, but a simple integer.
```
 "responses": {
          "200": {
            "description": "Success",
            "content": {
              "application/json": {
                "schema": {
                  "type": "integer",
                  "format": "int32"
                }
              },
              "text/json": {
                "schema": {
                  "type": "integer",
                  "format": "int32"
                }
              }
            }
          }
        }
```

However at the moment the generated javascript seems to assume it'll be a property of the response
```
let jsonData = {};
try {jsonData = pm.response.json();}catch(e){}

// Set property value as variable
const _res = jsonData?. ;

// pm.collectionVariables - Set newId as variable for jsonData. 
if (_res !== undefined) {
   pm.collectionVariables.set("newId", jsonData. );
   console.log("- use {{newId}} as collection variable for value",jsonData. );
} else {
   console.log('INFO - Unable to assign variable {{newId}}, as jsonData.  is undefined.');
};
```
Generated from this (empty string skips generation):
```
"assignVariables": [
        {
            "openApiOperation": "POST::/*",
            "collectionVariables": [
                {
                    "responseBodyProp": " ",
                    "name": "newId"
                }
            ]
        }
    ]
```

EDIT: Upon further testing, I'd probably need to test for 2XX before assigning the JSON response as well. I'm now using the ``portmanReplacements`` to do this.

**Comment by thim81** [comment]
hi @Arch0rr 

That is a good catch, looking at the code we do not yet support assigning the root element of a response.
We will investigate it further and keep you posted.

**Comment by thim81** [comment]
hi @Arch0rr 
We released Portman 1.20, which contains the new feature to store the root of the response as a variable.
